dietpi/dietpi-software

```diff
@@ -10929,7 +10929,7 @@ _EOF_
 
 			# Service: https://github.com/pymedusa/Medusa/blob/master/runscripts/init.systemd
 			G_EXEC cp /mnt/dietpi_userdata/medusa/runscripts/init.systemd /etc/systemd/system/medusa.service
-			G_EXEC sed -i 's/^[[:blank:]]Group=/#Group=/' /etc/systemd/system/medusa.service
```

Comment:
An Asterisk is missing there, it should have been:
```
G_EXEC sed -i 's/^[[:blank:]]*Group=/#Group=/' /etc/systemd/system/medusa.service
```
This catches every possible case where an active `Group` is in place, since leading spaces are allowed in systemd units.

Reply:
ah ok, didn't know that. Still bash scripting is something fare away 🤣

Reply:
That is more about regex, `sed` and `grep` syntax 😄.